apps/suite-launch/test-presets/golden-path-v01-11-2023/fluent-bit.env

```diff
@@ -0,0 +1,5 @@
+# Fargate was giving errors when loading environment variables.
+# ResourceInitializationError: failed to download env files: file download command: non empty error stream: service call has been retried 5 time(s): RequestCanceled: request context canceled caused by: context deadline exceeded.
```

Comment:
lack of permissions in execution role may be?

Reply:
Yes. I think it was actually a security group related error. We might be able to add this back in the future if we want to use environment variables.